src/social_db/permissions/grant_write_access/sign_as/mod.rs

```diff
@@ -0,0 +1,169 @@
+use std::str::FromStr;
+
+use color_eyre::eyre::ContextCompat;
+use inquire::{CustomType, Select};
+
+#[derive(Debug, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(input_context = super::storage_deposit::ExtraStorageDepositContext)]
+#[interactive_clap(output_context = SignerContext)]
+pub struct Signer {
+    #[interactive_clap(skip_default_input_arg)]
+    /// What is the signer account ID?
+    signer_account_id: near_cli_rs::types::account_id::AccountId,
+    #[interactive_clap(named_arg)]
+    /// Select network
+    network_config: near_cli_rs::network_for_transaction::NetworkForTransactionArgs,
+}
+
+#[derive(Clone)]
+pub struct SignerContext {
+    config: near_cli_rs::config::Config,
+    widgets: Vec<String>,
+    permission_key: crate::common::PermissionKey,
+    extra_storage_deposit: near_cli_rs::common::NearBalance,
+    signer_account_id: near_primitives::types::AccountId,
+}
+
+impl SignerContext {
+    pub fn from_previous_context(
+        previous_context: super::storage_deposit::ExtraStorageDepositContext,
+        scope: &<Signer as interactive_clap::ToInteractiveClapContextScope>::InteractiveClapContextScope,
+    ) -> color_eyre::eyre::Result<Self> {
+        Ok(Self {
+            config: previous_context.config,
+            widgets: vec![format!("{}/widget", scope.signer_account_id)],
```

Comment:
1. Rename the `widgets` field to `social_db_keys`
2. Make the `widget` (inside the `format!`) configurable - make it a positional argument to the `grant-write-access` subcommand: `socialdb permissions grant-write-access "socialdb-key-prefix" to-...` (when someone would need to allow write access to NEAR Social widgets, they would use `socialdb permissions grant-write-access "widget" to-...`)

Reply:
Done